wpa_supplicant/wpa_supplicant.c

```diff
@@ -905,6 +905,22 @@ void wpa_supplicant_reinit_autoscan(struct wpa_supplicant *wpa_s)
 	}
 }
 
+static void dhcp_war(struct wpa_supplicant *wpa_s)
```

Comment:
For an outsider it might not be clear what is this workaround for, would be nice to add comment about the problem.